App.js

```diff
@@ -0,0 +1,133 @@
+import { Console } from "@woowacourse/mission-utils";
+
+/**
+ * @enum {string} 계산기 문구 및 구분자 환경변수를 정의하는 열거형입니다.
+ * @property {string} INPUT_PROMPT 덧셈할 문자열을 입력받는 프롬프트 메시지
+ * @property {string} OUTPUT_PROMPT 결과를 출력하는 프롬프트 메시지
+ * @property {string} DEFAULT_SEPARATOR 기본 구분자 (콤마, 콜론)
+ * @property {string} CUSTOM_SEPARATOR 커스텀 구분자(//, \n 사이 문자)
+ */
+const CALCULATOR_VARIABLES ={
+
+    INPUT_PROMPT : '덧셈할 문자열을 입력해주세요.\n', 
+    OUTPUT_PROMPT : '결과 : ',
+    DEFAULT_SEPARATOR : '[,:]',
+    CUSTOM_SEPARATOR : '//(.)\\\\n'

```

Comment:
아래에서 `CUSTOM_SEPARATOR` 를 사용할 때 마다 `new RegExp()`로 `RegExp` 객체를 만들어주시는데, 아래처럼 정규식 리터럴로 표현한다면 아래에서 매번 객체를 만들어주지 않아도 됩니다!
```suggestion
    CUSTOM_SEPARATOR : /\/\/(.)\\n/
```